src/transaction/constants.ts

```diff
@@ -1,3 +1,6 @@
-export const SEND_CALLS_NOT_SUPPORTED_ERROR = 'SEND_CALLS_NOT_SUPPORTED_ERROR';
+export const ERROR_CODE = 'WRITE_TRANSACTIONS_ERROR';
```

Comment:
Need:

`WRITE_TRANSACTION_ERROR` and `WRITE_TRANSACTIONS_ERROR` for `writeContract` and `writeContracts`.